pom.xml

```diff
@@ -4624,6 +4625,12 @@
         <version>${version.javax.xml.stream.stax}</version>
       </dependency>
 
+      <dependency>
+        <groupId>link.bek.tools</groupId>
+        <artifactId>issue-keeper-junit</artifactId>
+        <version>${version.link.bek.tools.issue-keeper-junit}</version>
+      </dependency>
```

Comment:
Out of curiosity, can you explain why this dependency is needed? I guess I haven't seen it being directly used.

Reply:
This was all probably due a missing rebase, I will update this PR in a minute